coop_print_badge/__manifest__.py

```diff
@@ -0,0 +1,35 @@
+# Copyright (C) 2017-Today: La Louve (<http://www.lalouve.net/>)
+# @author: Julien Weste (julien.weste@akretion.com)
+# Copyright (C) 2019-Today: Druidoo (<https://www.druidoo.io>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    'name': 'Coop - Print Badge',
+    'version': '12.0.1.0.0',
+    'category': 'Custom',
+    'summary': 'Print partner\'s badge',
+    'author': 'La Louve',
+    'website': 'http://www.lalouve.net',
+    'depends': [
+        'base',
+        'web',
+        'barcodes_generator_abstract',
+        'barcodes_generator_partner',
+        'barcodes_generator_product',
+        'coop_shift',
+        'coop_membership',
+        'coop_capital_certificate',
```

Comment:
```suggestion
        'coop_membership',
        'coop_capital_certificate',
```

We don't need the other dependancies because:

- `web`, `base` are already depepnded by the other modules
- `barcode_generator_*` are already depended by coop_membership and coop_shift
- `coop_shift` is already depended by `coop_membership`